deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestDepositRecordJob.java

```diff
@@ -95,7 +99,20 @@ public void runJob() {
 				depositRecord.addManifest(new File(uri), "text/plain");
 			}
 
-			// TODO add references to deposited objects 
+			// Add references to deposited objects 
+			Bag depositBag = dModel.getBag(getDepositPID().getRepositoryPath());
+			List<Resource> children = new ArrayList<Resource>();
+			DepositGraphUtils.walkObjectsDepthFirst(depositBag, children);
+			
+			Model triples = ModelFactory.createDefaultModel();
+			List<Statement> relations = new ArrayList<Statement>();
+			for (Resource child : children) {
+				relations.add(triples.createStatement(depositRecord.getResource(),
```

Comment:
I don't think you need to go through the intermediate `relations` array here.  Prior to this loop I'd suggest doing a `triples.createResource(depositUri)` and then have the loop directly add to that resource.